src/xenia/kernel/xam/apps/xlivebase_app.cc

```diff
@@ -30,12 +30,14 @@ X_HRESULT XLiveBaseApp::DispatchMessageSync(uint32_t message,
   switch (message) {
     case 0x0005008C: {
       // Called on startup of blades dashboard v1888 to v2858
-      XELOGD("XLiveBaseUnk5008C, unimplemented");
+      XELOGD("XLiveBaseUnk5008C({:08X}, {:08X}), unimplemented", buffer_ptr,
+             buffer_length);
       return X_E_FAIL;
     }
     case 0x00050094: {
       // Called on startup of blades dashboard v4532 to v4552
-      XELOGD("XLiveBaseUnk50094, unimplemented");
+      XELOGD("XLiveBaseUnk50094({:08X}, {:08X}), unimplemented", buffer_ptr,
+             buffer_length);
       return X_E_FAIL;
     }
     case 0x00058003: {
```

Comment:
buffer_ptr, buffer_length. maybe add note they're always 0, 0

Reply:
When I last checked they returned (7015E380, 00000028)

Reply:
sorry i thought you meant 50094

Reply:
Should I have it return X_ONLINE_E_LOGON_NO_NETWORK_CONNECTION or just leave it as is?

Reply:
I suppose so since it's more correct than the current return value. However, you may as well add a few `X_ONLINE` codes to `xbox.h`. 

Just copy them from here.
https://github.com/AdrianCassar/xenia-canary/commit/3582698ce88793fae3b76ff2dcd8a5b3de002bfc#diff-6bbb739f532ac5500774a85bf50579573d860b91c27ca5019eae7197ca8b3eb3